src/Microsoft.Framework.NullLogging.Internal/NullLoggerFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.Framework.Internal
```

Comment:
This needs to be a separate project, take a look on how the other common utilities are.